pkg/blob/controllerserver.go

```diff
@@ -691,11 +698,17 @@ func (d *Driver) CreateBlobContainer(ctx context.Context, subsID, resourceGroupN
 			if getErr != nil {
 				return true, getErr
 			}
+			container.Metadata = metadata
 			_, err = container.CreateIfNotExists(&azstorage.CreateContainerOptions{Access: azstorage.ContainerAccessTypePrivate})
 		} else {
+			metadataPtr := make(map[string]*string)
+			for k, v := range metadata {
+				metadataPtr[k] = to.Ptr(v)
+			}
 			blobContainer := armstorage.BlobContainer{
 				ContainerProperties: &armstorage.ContainerProperties{
 					PublicAccess: to.Ptr(armstorage.PublicAccessNone),
+					Metadata:     metadataPtr,
```

Comment:
let's add a const tag: `createdBy`: driverOptions.DriverName in metadata, don't need to expose to the external user

Reply:
added it

Reply:
fix it to not expose to the cx